src/main/java/com/google/api/generator/gapic/protoparser/Parser.java

```diff
@@ -663,6 +697,14 @@ private static boolean isMapType(Descriptor messageDescriptor) {
                       /* protoPakkage */ protoMethod.getFile().getPackage(),
                       serviceDescriptor.getName(),
                       protoMethod.getName());
+      boolean operationPollingMethod = false;
+      if (protoMethod.getOptions().hasExtension(ExtendedOperationsProto.operationPollingMethod))
```

Comment:
Please never use if statements without `{}`. If you have ever used them like that elswere, please add curly brackets there too.